src/libraries/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -316,28 +327,28 @@ public partial class XmlDocument : System.Xml.XmlNode
+        public virtual System.Xml.XmlElement CreateElement(string? prefix, string localName, string? namespaceURI) { throw null; }
         public virtual System.Xml.XmlEntityReference CreateEntityReference(string name) { throw null; }
-        public override System.Xml.XPath.XPathNavigator CreateNavigator() { throw null; }
-        protected internal virtual System.Xml.XPath.XPathNavigator CreateNavigator(System.Xml.XmlNode node) { throw null; }
-        public virtual System.Xml.XmlNode CreateNode(string nodeTypeString, string name, string namespaceURI) { throw null; }
-        public virtual System.Xml.XmlNode CreateNode(System.Xml.XmlNodeType type, string name, string namespaceURI) { throw null; }
-        public virtual System.Xml.XmlNode CreateNode(System.Xml.XmlNodeType type, string prefix, string name, string namespaceURI) { throw null; }
+        public override System.Xml.XPath.XPathNavigator? CreateNavigator() { throw null; }
+        protected internal virtual System.Xml.XPath.XPathNavigator? CreateNavigator(System.Xml.XmlNode node) { throw null; }
+        public virtual System.Xml.XmlNode CreateNode(string nodeTypeString, string name, string? namespaceURI) { throw null; }
+        public virtual System.Xml.XmlNode CreateNode(System.Xml.XmlNodeType type, string name, string? namespaceURI) { throw null; }
+        public virtual System.Xml.XmlNode CreateNode(System.Xml.XmlNodeType type, string? prefix, string name, string? namespaceURI) { throw null; }
         public virtual System.Xml.XmlProcessingInstruction CreateProcessingInstruction(string target, string data) { throw null; }
-        public virtual System.Xml.XmlSignificantWhitespace CreateSignificantWhitespace(string text) { throw null; }
-        public virtual System.Xml.XmlText CreateTextNode(string text) { throw null; }
-        public virtual System.Xml.XmlWhitespace CreateWhitespace(string text) { throw null; }
-        public virtual System.Xml.XmlDeclaration CreateXmlDeclaration(string version, string encoding, string standalone) { throw null; }
-        public virtual System.Xml.XmlElement GetElementById(string elementId) { throw null; }
+        public virtual System.Xml.XmlSignificantWhitespace CreateSignificantWhitespace(string? text) { throw null; }
+        public virtual System.Xml.XmlText CreateTextNode(string? text) { throw null; }
+        public virtual System.Xml.XmlWhitespace CreateWhitespace(string? text) { throw null; }
+        public virtual System.Xml.XmlDeclaration CreateXmlDeclaration(string version, string? encoding, string? standalone) { throw null; }
+        public virtual System.Xml.XmlElement? GetElementById(string elementId) { throw null; }
         public virtual System.Xml.XmlNodeList GetElementsByTagName(string name) { throw null; }
         public virtual System.Xml.XmlNodeList GetElementsByTagName(string localName, string namespaceURI) { throw null; }
```

Comment:
`string namespaceURI` is correct here as it will throw but I think this is incidental and due to bug in the code, I think we should ideally fix the bug and change annotation (not in this PR)